content/en/blog/_posts/2021-0420-cncf-dan.md

```diff
@@ -0,0 +1,196 @@
++++ 
+title = "Thank You Dan Kohn" 
+date = 2021-03-19 
+author = ["Riaan Kleinhans"]
+lastmod = "Fri Mar 06 17:06:13 NZDT 2021" 
+summary = "He believed in ii, and in all of us." 
++++
```

Comment:
```suggestion
---

**Author**: Riaan Kleinhans
```

Reply:
Just to put it all together (sometimes I find the line by line suggestions a bit confusing), I got the page to render locally with this front matter and author line:

```
--- 
layout: blog
title: "Thank You Dan Kohn" 
date: 2021-03-19 
lastmod: "Fri Mar 06 17:06:13 NZDT 2021" 
slug: thank-you-dan-kohn
summary: "He believed in ii, and in all of us." 
---

**Author**: Riaan Kleinhans

```

I'm not sure that you need the `lastmod` or `summary` lines though.

Reply:
I found the blog guidelines section [regarding frontmatter formatting](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#technical-considerations-for-submitting-a-blog-post).

Note, the file name should also be changed to follow the format `YYYY-MM-DD-Your-Title-Here.md` as well.